src/test.jl

```diff
@@ -59,25 +68,27 @@ macro test_throws(logger, extype, expr)
     end
 end
 
-mutable struct TestHandler{F, O} <: Handler{F, O}
+mutable struct TestHandler{F, O, T} <: Handler{F, O}
```

Comment:
Why a parameter? I don't think there's a real penalty for dynamic dispatch on `occursin_msg`.